core/src/main/scala/org/http4s/Uri.scala

```diff
@@ -438,8 +469,11 @@ object Uri {
           )
       }
 
-    implicit val eq: Eq[Path] = Eq.fromUniversalEquals[Path]
-    implicit val semigroup: Semigroup[Path] = (a: Path, b: Path) => a.concat(b)
+    implicit val http4sInstancesForPath: Order[Path] with Semigroup[Path] =
+      new Order[Path] with Semigroup[Path] {
+        def compare(x: Path, y: Path): Int = x.compare(y)
+        def combine(x: Path, y: Path): Path = x.concat(y)
+      }
```

Comment:
This is new, but not exciting.  There probably ought to be a `Hash[Path]` here. Either @isomarcte didn't include it because it used to be a type alias, or I misplaced it because I'm juggling too damned many branches.

Reply:
> @isomarcte didn't include it because it used to be a type alias

Yeah, since it is just `String` on series/0.21, it wasn't needed to implement the Uri instance.